### ISSUE TITLE
Missing folders in workspaces cause Cmd+P to stop working in remote setup

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#94170]
Issue Type: <b>Bug</b>

On a remote ssh box:

1) Create a folder structure:
```
 <parentdir>
   testdir1/f1.js
   testdir2/f2.js
```
2) Create `<parentdir>/workspace.code-workspace` with the following content:
```
{
	"folders": [
		{
			"path": "testdir1"
		},
		{
			"path": "testdir2"
		}
	],
	"settings": {}
}
```
3) Open the workspace file from your local box
-> <kbd>Cmd+P</kbd>, they type `f`: both files show in the quick pick
4) Close window
5) On the remote machine remove/rename `testdir2`
6) Open the workspacefile from your local box
-> <kbd>Cmd+P</kbd>, they type `f`: no results show in the quick pick

The same setup works without issues in a local setup.

Originally reported at https://github.com/MicrosoftDocs/vsonline/issues/327 and confirmed.

VS Code version: Code - Insiders 1.44.0-insider (20b88fa0694cae8e46ffd305a1450ba58ae6065b, 2020-04-01T08:34:11.087Z)
OS version: Darwin x64 18.7.0
Remote OS version: Linux x64 5.0.0-1022-azure
Remote OS version: Linux x64 5.0.0-1022-azure

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.21GB free)|
|Process Argv|-psn_0_5776770|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: az-devbox|
|OS|Linux x64 5.0.0-1022-azure|
|CPUs|Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz (4 x 2294)|
|Memory (System)|15.64GB (2.95GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: az-devbox|
|OS|Linux x64 5.0.0-1022-azure|
|CPUs|Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz (4 x 2294)|
|Memory (System)|15.64GB (2.95GB free)|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
vsc-material-theme|Equ|32.5.0
vsc-material-theme-icons|equ|1.1.4
theme-monokai-pro-vscode|mon|1.1.15
vscode-docker|ms-|1.0.0
remote-ssh-edit-nightly|ms-|2020.3.44100
remote-ssh-nightly|ms-|2020.3.44100
remote-wsl|ms-|0.44.2
azure-account|ms-|0.8.8
js-debug-companion|ms-|0.0.4
vsonline|ms-|1.0.1889
material-icon-theme|PKi|4.0.1
vscode-ruby|win|0.27.0
gitlens|eam|10.2.1
vscode-pull-request-github-insiders|Git|2020.4.511
vscode-github-issue-notebooks|ms-|0.0.2
vscode-typescript-tslint-plugin|ms-|1.2.3

(3 theme extensions excluded)

</details>

**Comment by totszwai** [comment]
@roblourens which version have this fix?? 1.45.0 is broken right now.

**Comment by roblourens** [comment]
Only Insiders
